cmd/compose

```diff
@@ -188,23 +188,25 @@ gen_certs() {
     ####
     # Fresh start
     #### 
-    TEMP_CERT_PATH="${HOME}/docker_certs"
-    rm -rf ${TEMP_CERT_PATH}
-    mkdir -p ${TEMP_CERT_PATH}
+    TEMP_CERT_PATH=$(mktemp -d tmp.XXXXXXXXXX) || { echo "Failed to create temp file"; exit 1; }
+    TEMP_DOCKER_MACHINE_CERTS=$(mktemp -d) || { echo "Failed to create temp file"; exit 1; }
```

Comment:
"Failed to create temp file" - it's about folder right? "Failed to create temp folder"